config/ec2/config.go

```diff
@@ -237,6 +237,12 @@ func Configure(p *config.Provider) {
 			RefFieldName:      "SecurityGroupRefs",
 			SelectorFieldName: "SecurityGroupSelector",
 		}
+		r.LateInitializer = config.LateInitializer{
+			IgnoredFields: []string{
+				"ingress",
```

Comment:
I think we should remove these two fields from schema since the are already represented by another CRD, which is a case mentioned [in XRM](https://github.com/crossplane/crossplane/blob/master/design/one-pager-managed-resource-api-design.md#high-fidelity).

Reply:
fixed - removed in both fields 
but both fields present in status - think this is okay for observe in sg